src/features/calendar/components/index.tsx

```diff
@@ -70,7 +70,7 @@ const Calendar = () => {
       {
         pathname: undefined,
         query: {
-          campId: campId,
+          ...(campId && { campId: campId }),
```

Comment:
This syntax is crazy (but effective). 😊

Reply:
If someone had shown to me like two years ago that I would write and understand this, I would have laughed in their face 😂